packages/realm/src/Realm.ts

```diff
@@ -1963,17 +1963,33 @@ declare global {
   }
 }
 
+function getCallstack() {
+  try {
+    throw new Error("Finding calling stack");
+  } catch (err) {
+    const stack = err instanceof Error ? err.stack || "" : "";
+    return stack
+      .split("\n")
+      .map((line) => line.trim())
+      .filter((line) => line.startsWith("at"))
+      .splice(1); // Skipping this function
```

Comment:
Did you test that on all of node, hermes, and JSC? I know the stack format can differ between engines.

Reply:
Nope ... I thought I wanted to add this with support for Node.js first and then add to it later (because it'll take time to test across runtime).

Anyways, thanks for the reminder. I'll likely test this tomorrow 🤞

Reply:
This is the output on iOS with Hermes (arguably not the most useful because source-maps aren't applied by default 🤔):

```
Your app is relying on a Realm global, which will be removed in realm-js v13, please update your code to ensure you import Realm via a named import:

import { Realm } from "realm"; // For ES Modules
const { Realm } = require("realm"); // For CommonJS

This is happening at anonymous (http://192.168.1.101:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=org.reactjs.native.example.RealmReactNativeTests:162740:5)
```

Reply:
When revisiting this, I came up with a better strategy: https://github.com/realm/realm-js/pull/5660